docs/user_guide/custom_operations.md

```diff
@@ -69,21 +69,36 @@ container.
 
 ## TensorFlow
 
-Tensorflow allows users to [add custom
+TensorFlow allows users to [add custom
 operations](https://www.tensorflow.org/guide/create_op) which can then
-be used in TensorFlow models. By using LD_PRELOAD you can load your
-custom TensorFlow operations into Triton. For example, assuming your
-TensorFlow custom operations are compiled into libtfcustom.so,
+be used in TensorFlow models. You can load your custom TensorFlow operations
+into Triton in two ways: 
+* At runtime, by listing them in your model configuration.
+* At load time, by using LD_PRELOAD.
```

Comment:
I don't think calling LD_PRELOAD load time trick is accurate. I believe LD_PRELOAD is server launch time trick while model config is model load time trick. Am I mistaken?

Reply:
That's probably more intuitive wording for users, you're right. I've updated the documentation.